### ISSUE TITLE
cylc user guide: more advanced cycling info

### ISSUE THREAD

**Issue by benfitzpatrick** [cylc/cylc-doc#70]
We could do with more information about advanced cycling features such as:
- min(...)
- truncation syntax (e.g. `T-00` for `start at the next 00 minutes`)

and anything else that comes to mind.

**Comment by wxtim** [comment]
`[[[<repetitions>/<date1>/<date2>]]]` now covered:

https://cylc.github.io/cylc-doc/latest/html/user-guide/writing-workflows/scheduling.html#format-1-r-limit-datetime-datetime

**Comment by wxtim** [comment]
@oliver-sanders  I'm not clear whether `[[[<repetitions>/<duration>/(^|$)(+|-)<duration>]]]` is format 4 in the documentation. If it is I think this is closable.

**Comment by oliver-sanders** [comment]
I think the last bit differs:

```
<repetitions>/<duration>/(^|$)(+|-)<duration>
(^|$)(+|-)<duration>
```

E.G.

```
^+PT1H
```

Vs:

```
R[limit?]/[interval]/[datetime]
[datetime]
```

E.G:

```
2000-01-01
```

**Comment by MetRonnie** [comment]
`^+PT1H` is still a datetime (ICP + 1 hour). There is an example in the docs:
> `R1//+P0D  # Run once at the final cycle point`

**Comment by oliver-sanders** [comment]
Yes, but it's a different (and non-iso8602) way of writing it which requires documentation.

**Comment by oliver-sanders** [comment]
That doesn't cover this particular example.

We could probably just do with a clarification of where this extension to ISO8601 is valid and slip a couple more examples in.
